onnxruntime/contrib_ops/cuda/math/bias_dropout_impl.cu

```diff
@@ -79,6 +79,54 @@ __global__ void BiasDropoutKernel(
   }
 }
 
+
+template <typename T, bool has_residual>
+__global__ void BiasDropoutSameShapeKernel(
```

Comment:
Do we need to introduce a new kernel for this mode? 
Can we introduce a new template parameter, say bool has_same_shape_bias?